apps/user/src/layouts/FormLayout.tsx

```diff
@@ -31,7 +31,7 @@ export default FormLayout;
 
 const StyledFormLayout = styled.section`
     width: 100%;
-    height: 100vh;
+    height: fit-content;
```

Comment:
얘도 바닥에 붙네요.

```suggestion
    height: fit-content;
    margin-bottom: 240px;
```

다른 페이지와 마찬가지지만 마진 바텀을 주는게 좋을 것 같습니다.